java_tests/src/test/java/org/lsst/sal/SAL_AddedGenerics.java

```diff
@@ -3,36 +3,35 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-/**
- * A few tests require CSC that doesn't use generics, so we use Script.
- */
-public class SAL_ScriptTest {
+public class SAL_AddedGenericsTest {
 
     /**
-     * Test a SAL component that does not have csc in AddedGenerics.
+     * Test topics for a SAL component that does not have "csc" in AddedGenerics.
+     *
+     * Use Script: one of the few SAL component that is not a CSC.
      */
     @Test
-    public void testNoCscGenerics() {
+    public void testNoCsc() {
         Class<?> cls = SAL_Script.class;
         try {
-            cls.getMethod("Test_command_enterControlC", cls);
+            cls.getMethod("Script_command_enableC", cls);
             Assert.fail("A NoSuchMethodException should have been raised.");
         } catch (NoSuchMethodException e) {
             Assert.assertNotNull(e);
         }
         try {
-            cls.getMethod("Test_logevent_summaryStateC", cls);
+            cls.getMethod("Script_logevent_summaryStateC", cls);
             Assert.fail("A NoSuchMethodException should have been raised.");
         } catch (NoSuchMethodException e) {
             Assert.assertNotNull(e);
         }
     }
 
     /**
-     * Test that enterControl is not present for Test.
+     * Test that a standard CSC does not have the "enterControl" command.
      */
     @Test
-    public void testNoEnterControl() {
+    public void testNoCsc() {
```

Comment:
Now the class has two methods called `testNoCsc` which is not allowed in Java. Also, this method was called the same as in the python unit test so why rename it? Please make sure that this method is called the same as the corresponding python unit test.

Reply:
Thanks for catching that. Stupid error on my part.

I simplified the name of one of the corresponding Python test methods and renamed both Java functions to match the Python.